lib/spam-list.js

```diff
@@ -0,0 +1,8 @@
+const dbs = require('./dbs')
+
+module.exports = async function getSpamList () {
+  const { gk } = dbs()
+  return gk.get('spam')
+  .then(spamDoc => spamDoc.spam)
+  .catch(() => {})
```

Comment:
💡 What do you think about returning an empty array when either the database connect did not work or the doc could not be found? That way, you don't interrupt the program for something that is not a critical error condition.

(Might be better to do it on the call side tho)

Reply:
(sorry for this drive by thing, after a week of staying in bed with a cold I just started to work off my inbox and this was on top :D)

Reply:
good idea, thanks!

Reply:
or rather, make it so that if an error occurs, we don’t overwrite things in index.js, so at worst we block the old spammer list.